pkgs/development/python-modules/scikit-learn/default.nix

```diff
@@ -66,10 +66,23 @@ buildPythonPackage rec {
 
   doCheck = !stdenv.isAarch64;
 
-  # Skip test_feature_importance_regression - does web fetch
-  disabledTests = [ "test_feature_importance_regression" ];
+  disabledTests = [
+    # Skip test_feature_importance_regression - does web fetch
+    "test_feature_importance_regression"
+  ];
 
-  pytestFlagsArray = [ "-n" "$NIX_BUILD_CORES" "--pyargs" "sklearn" ];
+  pytestFlagsArray = [
+    # verbose build outputs needed to debug hard-to-reproduce hydra failures
+    "-v"
+    "--pyargs" "sklearn"
+    # NuSVC memmap tests causes segmentation faults in certain environments
+    # (e.g. Hydra Darwin machines) related to a long-standing joblib issue
+    # (https://github.com/joblib/joblib/issues/563). See also:
+    # https://github.com/scikit-learn/scikit-learn/issues/17582
+    "-k 'not (NuSVC and memmap)'"
+  ] ++ lib.optionals (!stdenv.isDarwin) [
+    "-n" "$NIX_BUILD_CORES"
+  ];
```

Comment:
Parallel testing worked fine for me on darwin. Did it not for you?

Reply:
It did not. See https://github.com/NixOS/nixpkgs/pull/122687#issuecomment-843324410

Reply:
I'm quite sure that the crash mentioned in the post you're referring to was due to the syntax error in `disabledTests` which is now resolved. The same crash was happening on linux by that time BTW. This was not a darwin specific issue.